src/prefect/cli/__init__.py

```diff
@@ -33,8 +33,12 @@ def make_user_config():
     with open(user_config_path, "w") as user_config:
         user_config.write(
             "# This is a user configuration file.\n"
-            "# Settings placed here will overwrite Prefect's defaults."
+            "# Settings placed here will overwrite Prefect's defaults.\n"
         )
+        config_loc = os.path.join(__file__, "../config.toml")
+        with open(config_loc) as defaults_file:
```

Comment:
```suggestion
        with open(config_loc, "r") as defaults_file:
```
While `"r"` is the default mode for open, it's standard practice to include it explicitly.